plutus-core-spec/plutus-core-specification.tex

```diff
@@ -906,25 +932,27 @@ \section{Built in types, functions, and values}
   the value \texttt{(error)}.
 \item We provide two versions of the division and remainder operations
   for integers.  These differ in their treatment of negative
-  arguments\footnote{The standard mathematical definition (the
-    so-called \textit{division algorithm}, or \textit{Euclidean
-      division}) is as follows: if $n,d \in \mathbb{Z}$ and $d \ne 0$
-    then there exist unique integers $q, r \in \mathbb{Z}$ such that
-    $n=qb+r$ and $0 \le r < \lvert d \rvert$; $q$ is the
-    \textit{quotient} and $r$ is the \textit{remainder}.  Neither of
-    our sets of operations implements this definition! For $d > 0$,
-    \texttt{divideInteger} and \texttt{modInteger} give the correct
-    results, but they can give negative remainders for $d<0$.  In any sensible
-  computation we'd probably have $d>0$, but it's worth being careful.}:
+  arguments.
+
+%%\footnote{The standard mathematical definition (the so-called
+%%\textit{division algorithm}, or \textit{Euclidean division}) is as
+%%follows: if $n,d \in \mathbb{Z}$ and $d \ne 0$ then there exist
+%%unique integers $q, r \in \mathbb{Z}$ such that $n=qb+r$ and
+%%$0 \le r < \lvert d \rvert$; $q$ is the \textit{quotient} and $r$
+%%is the \textit{remainder}.  Neither of our sets of operations
+%%implements this definition! For $d > 0$, \texttt{divideInteger}
+%%and \texttt{modInteger} give the correct results, but they can
+%%give negative remainders for $d<0$.  In any sensible computation
+%%we'd probably have $d>0$, but it's worth being careful.}:
```

Comment:
Is this supposed to be a code comment? If not, I say either delete it or don't!

Reply:
I've decided to double down and extend it.  The reason it's there is that I thought I'd better check exactly what these operations did and I was surprised that neither pair of operations did what I was expecting.  I've since discovered that the great majority of computer languages also fail to do what most mathematicians would regard as being the correct thing.  This is moderately subtle, so I think it's worth a note.